projects/1.16/assets/woot/woot/lang/zh_cn.json

```
@@ -1,120 +1,120 @@
 {
   "advancements.woot.capture_mob.description": "制造一个生物碎片，捕获一只绵羊并且完全编写碎片",
-  "advancements.woot.capture_mob.title": "众生皆伤",
+  "advancements.woot.capture_mob.title": "此过程中所有生物均受到伤害",
```

```suggestion
  "advancements.woot.capture_mob.title": "众生皆伤",
```
是进度名

---

这里原句梗的是No Animals Were Harmed，所以就翻成这样了

---

合理
原译是`没有动物受到伤害`，`此过程中`似乎用了 `No Animals Were Harmed in the Making of This Film` 的梗，不过模组里没有 `in the making of...`，所以
也许这样梗的辨识度会变低
```suggestion
  "advancements.woot.capture_mob.title": "所有生物均受到伤害",
```